openapi.yaml

```diff
@@ -29,6 +29,11 @@ paths:
                     type: "array"
                     items:
                       type: "object"
+                      additionalProperties: false
```

Comment:
Makes API much more strict; do not allow extra properties, and always require uuid, text and status columns to be returned.